src/utils/database.js

```diff
@@ -0,0 +1,26 @@
+import { Sequelize } from "sequelize";
+import { config } from "dotenv";
+config({ path: "./src/config.env" });
+
+const PROD_DB_PATH = process.env.PROD_DB_PATH;
+const ENV = process.env.ENV;
```

Comment:
Checking if the env variables exists or not is a nice practice.
Sometime adding a default value is also nice.